SETUP.sql

```diff
@@ -0,0 +1,20 @@
+drop table `weather_d_b`
```

Comment:
@carojaspy please take a look of how beego manage migrations?

Reply:
Already I added the migration to skip this Script in the commit  28271025909eb7f27844ccd847be54126824f153